Source/Diagnostics/FullDiagnostics.cpp

```diff
@@ -764,13 +764,16 @@ FullDiagnostics::InitializeBufferData (int i_buffer, int lev, bool restart ) {
         // Coarsen and refine so that the new BoxArray is coarsenable.
         ba.coarsen(m_crse_ratio).refine(m_crse_ratio);
 
+        // Box covering the extent of the user-defined diagnostic domain
+        amrex::Box const domain = ba.minimalBox();
```

Comment:
```suggestion
        amrex::Box domain = diag_box;
        domain.coarsen(m_crse_ratio).refine(m_crse_ratio);
```
It's cheaper to do it this way. `ba.minimalBox()` needs to loop over all boxes in the boxarray.